src/execution/window_segment_tree.cpp

```diff
@@ -278,7 +278,7 @@ unique_ptr<WindowAggregatorState> WindowCustomAggregator::GetLocalState() const
 }
 
 void WindowCustomAggregator::Evaluate(WindowAggregatorState &lstate, const idx_t *begins, const idx_t *ends,
-                                      Vector &result, idx_t count) const {
+                                      Vector &result, idx_t count, idx_t row_idx) const {
```

Comment:
How does `WindowCustomAggregator` handle exclude? A simple test is to use it with windowed `COUNT(*)`

Reply:
Currently ignoring the exclude clause. I'll add the implementation for `WindowCustomAggregator`.